starboard/android/apk/app/build.gradle

```diff
@@ -149,16 +149,16 @@ android {
         }
         // Add the directories symlinked by the CMake "cobalt_content" custom command.
         debug {
-            assets.srcDir "${buildDir}/intermediates/cxx/debug/cobalt_content"
+            assets.srcDir "${cobaltContentDir}"
         }
         devel {
-            assets.srcDir "${buildDir}/intermediates/cxx/devel/cobalt_content"
+            assets.srcDir "${cobaltContentDir}"
         }
         qa {
-            assets.srcDir "${buildDir}/intermediates/cxx/qa/cobalt_content"
+            assets.srcDir "${cobaltContentDir}"
         }
         release {
-            assets.srcDir "${buildDir}/intermediates/cxx/gold/cobalt_content"
+            assets.srcDir "${cobaltContentDir}"
```

Comment:
What are these changes for?

Reply:
To work around with the problem caused by gradle upgrade.